source/pool/contracts/Pool.sol

```diff
@@ -363,4 +368,34 @@ contract Pool is IPool, Ownable {
     bytes32 leaf = keccak256(abi.encodePacked(participant, score));
     return MerkleProof.verify(proof, root, leaf);
   }
-}
+
+/** @notice withdraw function that uses signature instead of claim
+  * @param signature signature from activate or any admin
+  * @param messageHash hash of token address, minimumAmount, recipient and nonce
+  * @param token address
+  * @param amount uint256
+  * @param recipient address
+  */
+  function withdrawWithSignature(
+    bytes memory signature,
+    bytes32 messageHash,
+    address token,
+    uint256 amount,
+    address recipient
+  ) external override multiAdmin returns (uint256) {
+    // verify signed hash has not been claimed
+    require(!claimed[messageHash][recipient], "CLAIM_ALREADY_MADE");
+    // to verify hash is signed by an admin, ECDSA.recover will throw error if signer is not recoverable
+    bytes32 ethSignedMessageHash = ECDSA.toEthSignedMessageHash(messageHash);
+    address signer = ECDSA.recover(ethSignedMessageHash, signature);
+    require(signer == msg.sender, "NOT_VERIFIED");
```

Comment:
This should require that signer is among the admins i.e. `admins[signer] == true`

Reply:
ok this is modified accordingly.